File: Corte1/Tarea 120826Entrega140826/lluvia.cpp
#include <iostream>
using namespace std;

int main() {
    int datosLluvia[9] = {0, 1, 1, 0, 1, 1, 1, 0, 1};
    int rachaMaxima = 0;
    int rachaActual = 0;

    for (int i = 0; i < 9; i++) {
        if (datosLluvia[i] == 1) {
            rachaActual++;
            if (rachaActual > rachaMaxima) {
                rachaMaxima = rachaActual;
            }
        } else {
            rachaActual = 0;
        }
    }

    cout << "La racha máxima de lluvia fue de " << rachaMaxima << " días." << endl;
    return 0;
}

File: Corte1/Tarea 120826Entrega140826/lluvia.py
rachaMaxima = 0
rachaActual = 0
datosLluvia = [0,1,1,0,1,1,1,0,1]
for i in datosLluvia:
    if i == 1:
        rachaActual += 1
        if rachaActual > rachaMaxima:
            rachaMaxima = rachaActual
    else:
        rachaActual = 0
print(f"La racha máxima de lluvia fue de {rachaMaxima} días.")